src/main/java/xyz/nucleoid/plasmid/game/ConfiguredGame.java

```diff
@@ -4,23 +4,31 @@
 import com.mojang.serialization.MapCodec;
 import net.minecraft.server.MinecraftServer;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.concurrent.CompletableFuture;
 
 public final class ConfiguredGame<C> {
     public static final Codec<ConfiguredGame<?>> CODEC = GameType.REGISTRY.dispatchStable(c -> c.type, ConfiguredGame::codecFor);
 
     private final GameType<C> type;
     private final C config;
+    private static List<ConfiguredGame<?>> openingGames = new ArrayList<>();
```

Comment:
Is there a reason why this is a list? A non-list based approach may be easier here 😅

Reply:
If two game opens before the GameWorld is open, the ConfiguredGame will not match the GameWorld.

Reply:
That's true- but a list doesn't seem like the most appropriate solution in this instance.

Reply:
What would you suggest then?

Reply:
A usecase for this feature would probably help in making that decision 😅

Reply:
I wanted to show more information in game-gui when choosing a game to join. And to get those info I need to be able to know the ConfiguratedGame being played in a GameWorld.

Reply:
What it looks like rn. ![image](https://user-images.githubusercontent.com/34719527/90076100-2e503100-dcff-11ea-92a8-e691ee1ed058.png)